src/Core/src/ImageSources/Android/Glide/MauiAppGlideModule.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Content;
+using Android.Graphics;
+using Android.Runtime;
+using Bumptech.Glide;
+using Java.IO;
+
+namespace Microsoft.Maui.BumptechGlide
+{
+	// This class name is very specific as it is directly loaded.
+	[Register("com.bumptech.glide.GeneratedAppGlideModuleImpl")]
+	public class MauiAppGlideModule : GeneratedAppGlideModule
+	{
+		public MauiAppGlideModule(Context context)
+		{
+		}
+
+		public override void RegisterComponents(Context context, Glide glide, Registry registry) => registry
+			.Prepend(GetClass<InputStream>(), GetClass<InputStream>(), new PassThroughModelLoader<InputStream>.Factory())
+			.Append(GetClass<FontImageSourceModel>(), GetClass<FontImageSourceModel>(), new PassThroughModelLoader<FontImageSourceModel>.Factory())
+			.Append(GetClass<FontImageSourceModel>(), GetClass<Bitmap>(), new FontImageSourceDecoder());
```

Comment:
We also have the font image sources here which means we can "decode" a font image and it fits in the pipeline as it were. It also fits in with the caching and all that, which means that they can be reused and reduces the images created.